src/pages/loading-waiting-states/spinner-overlays/index.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import {
+    AppBar,
+    Button,
+    Card,
+    Checkbox,
+    CircularProgress,
+    Divider,
+    FormControlLabel,
+    Hidden,
+    IconButton,
+    Toolbar,
+    Typography,
+} from '@material-ui/core';
+import { makeStyles, Theme, useTheme } from '@material-ui/core/styles';
+import MenuIcon from '@material-ui/icons/Menu';
+import { useDispatch } from 'react-redux';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import { eulaDetails } from './data';
+import * as Colors from '@pxblue/colors';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    container: {
+        padding: `${theme.spacing(2)}px ${theme.spacing(2)}px`,
+        maxWidth: '600px',
+        margin: '0 auto',
+        height: `calc(100vh - ${theme.spacing(8)}px)`,
+        [theme.breakpoints.down('xs')]: {
+            height: `calc(100vh - ${theme.spacing(7)}px)`,
+        },
+    },
+    toolbarGutters: {
+        padding: `0 ${theme.spacing(2)}px`,
+    },
+    eulaContainerStyle: {
+        height: `calc(100vh - 100px)`,
+        position: 'relative',
+        padding: 0,
+    },
+    loadingOverlayContainerStyle: {
+        display: 'flex',
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        width: '100%',
+        height: '100%',
+        zIndex: 4,
+        justifyContent: 'center',
+        alignItems: 'center',
+        flexDirection: 'column',
+    },
+    eulaDetailsStyle: {
+        flex: '1 1 0',
+        display: 'flex',
+        flexDirection: 'column',
+        height: '100%',
+    },
+    eulaDetailSectionStyle: {
+        overflow: 'auto',
+        display: 'flex',
+        flexDirection: 'column',
+        flex: '1 1 0',
+        padding: `${theme.spacing(3)}px`,
+    },
+    loadingContainerStyle: {
+        height: '100%',
+        paddingTop: `${theme.spacing(1)}px`,
+    },
+    overlayContainerStyle: {
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        width: '100%',
+        height: '100%',
+        background: Colors.white[200],
+        zIndex: 3,
+        opacity: 0.2,
+    },
+    btnContainerStyle: {
+        padding: `${theme.spacing(3)}px`,
+    },
+    reloadButtonStyle: {
+        width: '100%',
+    },
+    eulaContentStyle: {
+        paddingTop: `${theme.spacing(1)}px`,
+    },
+    eulaConfirmationCheckStyle: {
+        padding: `0 ${theme.spacing(3)}px`,
+    },
```

Comment:
Since these are all styles by definition, you can probably omit "style" from each of these classes.